### PR TITLE
Moved Config Resources and Missing Prune

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- moved config resources to new KEB chart,
- added missing `prune` annotation for Argo.
**Related issue(s)**

### CODE REVIEW

**Review by MarekMichali — APPROVED** [review]

File: resources/keb/values.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 global:
```
[the remark this answers is not in the document]
Comment (ralikio) [reply resources/keb/values.yaml:1]: Thank you for noticing. Images are not use in this portion. Deployments that actually used those images will be merged after the current PR. Thanks to your comment I seen that there is one commit missing. Please tak a look at the latest commit.

File: resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml [hunk 2]
```diff
@@ -63,14 +63,10 @@ spec:
           image: "{{ .Values.global.images.container_registry.path }}/{{ .Values.global.images.kyma_environment_broker.dir }}kyma-environment-broker:{{ .Values.global.images.kyma_environment_broker.version }}"
           imagePullPolicy: {{ .Values.deployment.image.pullPolicy }}
           env:
```
[the remark this answers is not in the document]
Comment (ralikio) [reply resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml:65]: KEB is not migrated in that stage.

File: resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml [hunk 3]
```diff
@@ -310,8 +310,6 @@ spec:
               value: "{{ .Values.avs.providerTagClassId }}"
             - name: APP_AVS_SHOOT_NAME_TAG_CLASS_ID
               value: "{{ .Values.avs.shootNameTagClassId }}"
-            - name: APP_AVS_EXTERNAL_TESTER_DISABLED
```
Comment (ralikio) [inline resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml:313]: Removal of duplicated environment. Please compare with dev.

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**